### PR TITLE
add CODEOWNERS

### PR DESCRIPTION
**Pull request by aiuto**
#138

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+# When someone opens a pull request that only modifies files in directory, only
+# component owners and not the global owner(s) will be requested for a review.
+
+# Syntax: https://help.github.com/articles/about-codeowners/
+
+# Global
+* @c-parsons
+
+/skydoc @davidzchen
```
Comment (c-parsons) [inline CODEOWNERS]: Does david still "own" this? I don't think he has worked on this in quite some time -- we might want to consider just having me down as owner for this as well. (We can get him to chime in, certainly :))

**Comment by aiuto** [comment]
+dzc   David? Do you want to get pull requests, or move on.

**Comment by c-parsons** [comment]
@davidzchen

**Comment by laurentlb** [comment]
Why do we need this file? David is not working on this (and hasn't for a long time).
I think @c-parsons and I can handle the requests.

**Comment by aiuto** [comment]
We want CODEOWNERS for all repositories in bazelbuild so that we can do
unified triage against them all.
And by "we" I mean not just the team today, but people added in the future.
It should be trivial for someone who does not know you or Chris to
auto-assign a PR without looking at code history. The goal is to get
institutional knowledge out of peoples' heads and into code.
We're doing this for all the repos.

**Comment by laurentlb** [comment]
> so that we can do unified triage against them all.

Why do you need to triage pull requests from this repository? Users don't need to assign the PRs themselves (actually, they can't).

Each repository should have a set of collaborators (with admin/write access). Do you want to duplicate this information?

**Comment by aiuto** [comment]
We (the developer experience team @jin @dslomov @iirina @irengrig ) are trying to get a unified policy for maintaining all of Bazel. In short the lifecycle of an issue: https://bazel.build/maintainers-guide.html#lifecycle-of-an-issue should apply to all the repos we maintain (and maybe anything under bazelbuild).

We would like the first tier triage on-call to be able to look at issues and pr's across project and nudge the lingering ones along. In the longer term, we'll do automation to make this happen. For now, we just want to make sure that it is trivial for the on-call to know who to assign things to.

Right now, each DevEx on-call has to have tacit knowledge of who owns what bits of code. Using
CODEOWNERS makes the knowledge explicit.  The collaborators is too broad. I think I have write access to this repo, but I won't know what to do with most PRs.

**Comment by jin** [comment]
> Each repository should have a set of collaborators (with admin/write access). Do you want to duplicate this information?

This information is not visible to users who are not collaborators of the repository (no settings tab).

File: CODEOWNERS (hunk 1 above)
Comment (aiuto) [reply CODEOWNERS]: SGTM

**Pull request closed without merging**